sale_coupon_multi_use/views/sale_coupon_views.xml

```diff
@@ -24,6 +25,7 @@
                     attrs="{'invisible': [('multi_use', '=', False)]}"
                 >
                     <tree>
+                        <field name="currency_program_id" invisible="1" />
```

Comment:
To move in your `sale_coupon_multi_currency` module: it's not linked to multi use.